src/orquestra/integrations/qiskit/conversions/_circuit_conversions.py

```diff
@@ -151,6 +158,26 @@ def _export_gate_via_mapping(gate, applied_qubit_indices, q_register, custom_nam
     return qiskit_cls(*qiskit_params), qiskit_qubits, []
 
 
+def _export_dagger_gate(
```

Comment:
[comment] I feel like this whole logic for raising exceptions if export failed got a bit out of hand. Though I don't think it's worth refactoring as a part of this PR, that would be to much hassle.
Could you make a ticket for refactoring it?

Reply:
For sure. It's probably slow too.